timestamp-client/src/test/java/com/palantir/timestamp/RateLimitedTimestampServiceTest.java

```diff
@@ -30,27 +29,24 @@
 public class RateLimitedTimestampServiceTest {
```

Comment:
I didn't really look at these tests beyond a sanity check, but as you say it's been proven in the field (and I'm using that as my basis for confidence).

Longer-term, we could look at extending Jepsen to support multithreaded timestamp clients (though I'd still propose keeping the current test with single-threaded clients as it's often easier to debug server issues with those).